includes/class-wcal-guest-ac.php

```diff
@@ -263,110 +263,113 @@ function save_data() {
 			// Insert record in guest table.
 			$billing_first_name = wcal_common::wcal_get_cart_session( 'billing_first_name' );
 
-			$billing_last_name = wcal_common::wcal_get_cart_session( 'billing_last_name' );
+			$billing_last_name         = wcal_common::wcal_get_cart_session( 'billing_last_name' );
+			$billing_email             = wcal_common::wcal_get_cart_session( 'billing_email' );
+			$billing_email_restriction = apply_filters( 'wcal_abandoned_cart_user_email', false, $billing_email );
 
 			$shipping_zipcode = '';
 			$billing_zipcode  = '';
+			if ( true !== $billing_email_restriction ) {
```

Comment:
We can simplify

`if ( true !== $billing_email_restriction ) {`

to 

`if ( ! $billing_email_restriction ) {`